src/tests/test_meta_blk_mgr.cpp

```diff
@@ -497,13 +497,16 @@ class VMetaBlkMgrTest : public ::testing::Test {
 
     void recover_with_on_complete() {
         // restart will cause recovery and callbacks will be triggered
+        ++m_restart_cnt;
         m_cb_blks.clear();
         restart_homestore();
     }
 
     void validate() {
         // verify received blks via callbaks are all good;
         verify_cb_blks();
+        m_write_sbs.swap(m_cb_blks);
```

Comment:
use `m_cb_blks` to replace `m_write_sbs`, as all the cookies saved in `m_write_sbs` is invalid after restart